src/game/main.cpp

```diff
@@ -81,14 +85,17 @@ extern HINSTANCE &g_applicationHInstance;
 #else
 #ifdef PLATFORM_WINDOWS
 HWND g_applicationHWnd;
+HGDIOBJ g_splashImage;
+HINSTANCE g_applicationHInstance;
+#endif
+#ifdef BUILD_WITH_SDL2
+// SDL_Window *g_applicationWindow;
```

Comment:
What is this for?

Reply:
no idea, i removed it. Probably a leftover from earlier